README.md

```diff
@@ -590,7 +588,34 @@ encountered.
 
 ### 5. Reporting Issues
 
-If you are having problems with HarbourBridge, please [submit an
-issue](https://github.com/cloudspannerecosystem/harbourbridge/issues).
+## Known Issues
+Please refer to the [issues section](https://github.com/cloudspannerecosystem/harbourbridge/issues)
+ on Github for a full list of known issues.
+
+### Schema Conversion
+
+- Loading dump files from SQL Server and DynamoDB is not supported
+- Schema Only Mode does not create foreign keys
+- Migration of constraints, functions and views is not supported
```

Comment:
IMO the term constraints seem a little confusing here because Fks and PKs are also constraints but they get migrated.

Reply:
Yes we should change this to check constraints maybe?

Reply:
Done